### PR TITLE
Feature/upgrade to django 1.10 wagtail 1.10

### PR DESCRIPTION
**Pull request by codiebeulaine**
Disclaimer: This PR has fallen victim to scope creep and is a lot larger than it should be.

In the PR we have moved molo profiles into molo core, we have updated from Django 1.9 to 1.10 and Wagtail 1.9 to 1.10

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony @alexmuller ready for review

**Comment by nathanbegbie** [comment]
@codiebeulaine if I grep for `docs/multi_site.png` I don't see it referenced any where. Do we need it?

**Comment by codiebeulaine** [comment]
@nathanbegbie no we do not

**Comment by nathanbegbie** [comment]
:+1:

File: molo/core/wagtail_hooks.py [hunk 1]
```diff
@@ -25,14 +25,15 @@
 from wagtail.wagtailadmin.wagtail_hooks import page_listing_more_buttons
 
 from molo.core.api import urls as molo_api_urls
+from molo.core import views as views
```
[the remark this answers is not in the document]
Comment (codiebeulaine) [reply molo/core/wagtail_hooks.py]: @nathanbegbie our views need to be callable

**Pull request merged**